libs/execution_context/kernel/execution_context_kernel.vcxproj

```diff
@@ -73,6 +73,7 @@
     <ClCompile Include="..\ebpf_maps.c" />
     <ClCompile Include="..\ebpf_native.c" />
     <ClCompile Include="..\ebpf_program.c" />
+    <ClCompile Include=".\string_opts.c" />
```

Comment:
why is this only in kernel and not in both kernel and user like the other files above this line?
Rtl* functions should work in both user and kernel so I don't see why you can't use a common implementation.

Reply:
I have found the fix, and it was dumb. I needed to add `_AMD64_;` to the `<PreprocessorDefinitions>` nodes in the vcxproj to be able to use NtStrSafe functions.